g2p_programs/models/managers/entitlement_manager.py

```diff
@@ -70,12 +74,33 @@ class DefaultCashEntitlementManager(models.Model):
         "res.currency", related="program_id.journal_id.currency_id", readonly=True
     )
 
+    # Transfer Fees
+    transfer_fee_pct = fields.Integer(
```

Comment:
@kneckinator is an integer enough?

Reply:
The variable name seems to tell me it is a percentage, so we should use decimal.

Reply:
@gonzalesedwin1123

Reply:
OK, I will change the field type to decimal

Reply:
Done